script.js

```diff
@@ -134,23 +134,27 @@ const renderTodoList = (todoListState, onDestroy) => {
     notifyStateUpdated();
   };
 
-  inputField.addEventListener('keydown', (e) => {
+  const escapeEmpty = () => inputField.value = "";
```

Comment:
From my perspective this function should be called `cleanField`. 
It's a minor,  I leave it up to you

Reply:
Yes, you are right. Renamed)